### DISCUSSION TITLE
Docker Based AGW Check in Fail on Orc8r  some time while deploy  using master Branch.

### DISCUSSION THREAD

**Discussion by Aniket621** [magma/magma#15351] [Q&A] [answered]
Hi all,

a) Any one trying to test and deploy Docker based AGW (master branch) release.
b) I have deployed AGW in docker enviroment and on Bare Metal all container are showing up.
c) AGW Check in successfully and  registered on Orc8r and status is showing good.
d) But some times we observe AGW check in fail and status is showing "BAD" on NMS but container all are working and up. 
e) In Magmad logs observe and gives error the  AGW Checkin fail due to Unknown client certificate and this certificate has expired. 
f) After reboot the AGW then it's status is changed on NMS and it's showing Good health.
e) Magmad container logs attached here with. 

![image](https://github.com/magma/magma/assets/104416203/76ca1670-475e-4f08-a3a8-f0f7c4261b5f)
[Magmad Container logs.txt](https://github.com/magma/magma/files/13597316/Magmad.Container.logs.txt)

Any help and Guidance would be appreciated.

**Comment by panyogesh** [comment, marked as answer]
This seems to be certificate related. Can you re-generate the certificates (In case of developer orc8r delete the .cache directory). Will update on the orc8r  integrated with docker agw status shortly.

**Comment by aniket021997** [comment]
Hi,
We tried dockerized agw (master branch) with orc8r, agw is able to check in successfully.
Please find magmad container logs.
![image](https://github.com/magma/magma/assets/95931489/cb9f02fd-1441-477e-863a-9a6167d72674)

[logs_magmad.txt](https://github.com/magma/magma/files/13654826/logs_magmad.txt)

**Reply by Aniket621** [reply]
Hi Aniket we have also successfully regsitered the AGW on NMS you can see in above logs pic but some time it's status change from Good to BAD and in that case status change after AGW reboot.

**Reply by panyogesh** [reply]
@Aniket621 : This will become very specific bugging. For now closing it. In case if the issue is persistent please raise an issue.
